experimenter/experimenter/experiments/admin.py

```diff
@@ -328,14 +329,20 @@ class NimbusExperimentAdmin(
 
 class NimbusFeatureConfigAdmin(NoDeleteAdminMixin, admin.ModelAdmin[NimbusFeatureConfig]):
     prepopulated_fields = {"slug": ("name",)}
-    list_filter = ("application", "read_only")
-    list_display = ("name", "application", "read_only")
+    list_filter = ("application",)
+    list_display = ("name", "application")
     exclude = ("read_only",)
 
     def has_change_permission(self, request, obj: Optional[NimbusFeatureConfig] = None):
         return obj and not obj.read_only
 
 
+class NimbusVersionedFeatureConfigAdmin(
+    NoDeleteAdminMixin, admin.ModelAdmin[NimbusVersionedSchema]
+):
+    pass
```

Comment:
(I'm not super familiar with the admin stuff) what is this class doing? Why do we need a second one for the versioned configs?

Reply:
The `NoDeleteAdminMixin` prevents model instances from being deleted in the admin panel. 
The `NimbusVersionedFeatureConfig` the `ModelAdmin` class for `NimbusVersionedSchema`s (the name of the class is wrong -- search and replace didnt get this one) that tells the django admin panel about this model and how to display its models and forms, etc.